app/gateway/changelog.md

```diff
@@ -5,17 +5,372 @@ no_version: true
+  [#9432](https://github.com/Kong/kong/pull/9432)
+- Increase the default value of `lua_regex_cache_max_entries`. A warning will be thrown
+  when there are too many regex routes and `router_flavor` is `traditional`.
+  [#9624](https://github.com/Kong/kong/pull/9624)
+- Add batch queue into the Datadog and StatsD plugins to reduce timer usage.
+  [#9521](https://github.com/Kong/kong/pull/9521)
+
+### PDK
+
+- Extend `kong.client.tls.request_client_certificate` to support setting
+  the Distinguished Name (DN) list hints of the accepted CA certificates.
+  [#9768](https://github.com/Kong/kong/pull/9768)
+
+### Plugins
+
+**New plugins:**
+- [**AppDynamics**](/hub/kong-inc/app-dynamics) (`app-dynamics`)
+  - Integrate Kong Gateway with the AppDynamics APM Platform.
+- [**JWE Decrypt**](/hub/kong-inc/jwe-decrypt) (`jwe-decrypt`)
+  - Allows you to decrypt an inbound token (JWE) in a request.
+- [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`)
+  - Validate HTTP requests and responses based on an OpenAPI 3.0 or Swagger API Specification.
+- [**SAML**](/hub/kong-inc/saml) (`saml`)
+  - Provides SAML v2.0 authentication and authorization between a service provider (Kong Gateway) and an identity provider (IdP).
+- [**XML Threat Protection**](/hub/kong-inc/xml-threat-protection) (`xml-threat-protection`)
+  - This new plugin allows you to reduce the risk of XML attacks by checking the structure of XML payloads. This validates maximum complexity (depth of the tree), maximum size of elements and attributes.
+
+**Updates to existing plugins:**
+
+- [**ACME**](/hub/kong-inc/acme/) (`acme`)
+  - Added support for Redis SSL, through configuration properties
+  `config.storage_config.redis.ssl`, `config.storage_config.redis.ssl_verify`,
+  and `config.storage_config.redis.ssl_server_name`.
+  [#9626](https://github.com/Kong/kong/pull/9626)
+
+- [**AWS Lambda**](/hub/kong-inc/aws-lambda/) (`aws-lambda`)
+  - Added `requestContext` field into `awsgateway_compatible` input data
+  [#9380](https://github.com/Kong/kong/pull/9380)
+
+- [**Authentication plugins**](/hub/#authentication):
+  - The `anonymous` field can now be configured as the username of the consumer.
+  This field allows you to configure a string to use as an “anonymous” consumer if authentication fails.
+
+- [**OpenTelemetry**](/hub/kong-inc/opentelemetry/) (`opentelemetry`)
+  - Added referenceable attribute to the `headers` field
+  that could be stored in vaults.
+  [#9611](https://github.com/Kong/kong/pull/9611)
+
+- [**Forward Proxy**](/hub/kong-inc/forward-proxy/) (`forward-proxy`)
+  - `x_headers` field added. This field indicates how the plugin handles the headers
+  `X-Real-IP`, `X-Forwarded-For`, `X-Forwarded-Proto`, `X-Forwarded-Host`, and `X-Forwarded-Port`.
+
+    The field should be set to one of the below and is default to "append":
+    - `append`: append information of this hop to those headers;
```

Comment:
The word "hop" seems out of place here

Reply:
Rephrased based on a bit of reading on hop-by-hop headers.